SudokuBLL/src/main/java/pkgGame/Sudoku.java

```diff
@@ -124,6 +126,15 @@ public static boolean isRegionBoundary(double dSize)
 			return false;
 		}
 	}
+	
+	private boolean isDifficultyMet(int iPossibleValues) {
+		
+		int difficulty = this.eGameDifficulty.getiDifficulty();
+		int a = 3;
+		int b = iPossibleValues;
+		return difficulty < Math.pow(a, b) + 300;
+		
+	}
```

Comment:
We've now been dunked on by the Professor that his 100, 500, 1000 scale for the enum doesn't work for 4x4 or other similar NxN Sudoku puzzles. We'll keep this since there's not enough time for us to meet to fix it.